hs_swat_modelinstance/templates/includes/application_scripts.html

```diff
@@ -0,0 +1,17 @@
+{% if debug %}
+<script type="text/javascript" src='{{ STATIC_URL }}js/underscore.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/json2.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.marionette.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone-relational.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/moment-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/jqcsrf.js'></script>
+{% else %}
+<script type="text/javascript" src='{{ STATIC_URL }}js/underscore-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/json2.js'></script>
```

Comment:
The existing resource types make good starting points but you need to include the assets you actually need. Not only are you clearly not using these libraries in this new type, but they don't seem to be in use for *any* types as far as I can tell. It looks like @pkdash checked at least some of them in originally. Perhaps he can shed some light?